cmd/nodeman.go

```
@@ -0,0 +1,721 @@
+const JWK_KEY_ISS = "iss"
+
+var enrollWorkdir       string
+var enrollCredsFilename string
+var renewDatakey        string
+var renewClientKey      string
+var renewCaCertOut      string
+var renewClientCertOut  string
+func init() {
+	EnrollCmd.Flags().StringVarP(&enrollCredsFilename, FLAG_ENROLL_CREDENTIALS, "c", "", "DNSTAPIR enrollment credentials")
+	EnrollCmd.Flags().StringVarP(&enrollWorkdir, FLAG_ENROLL_WORKDIR, "w", DIRNAME_DEFAULT_WORKDIR, "Directory for storing cryptographic material on disk")
+    EnrollCmd.MarkFlagRequired(FLAG_ENROLL_CREDENTIALS)
```

golangci-lint (1.64.8)

Error return value of `EnrollCmd.MarkFlagRequired` is not checked

(errcheck)